susemanager-utils/testing/automation/backend-unittest-pgsql.sh

```diff
@@ -1,7 +1,25 @@
 #! /bin/sh
+if [ -z ${PRODUCT+x} ];then
+    PRODUCT="Uyuni"
+fi
+
+while getopts 'P:h' c
```

Comment:
```suggestion
while getopts 'P:h' option
```
Maybe more readable name for the variable?